jumanji/environments/routing/connector/env.py

```diff
@@ -391,23 +372,3 @@ def action_spec(self) -> specs.MultiDiscreteArray:
             dtype=jnp.int32,
```

Comment:
In the docstring of `action_spec`, I spotted a reference to the environment being multi-agent. Could we adapt it by replacing "Since this is a multi-agent environment, the environment expects..." with "Since this is an environment with a multi-dimensional action space, it expects..." ?